app/report/Report.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+import { useState } from "react";
+import { useMemo } from "react";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import classNames from "classnames";
+import { ReportResidencyFrTaxesFr } from "./residency_fr_taxes_fr/ReportResidencyFrTaxesFr";
+
+export type CountryCode = "us" | "fr";
+export interface ReportProps {}
+
+export const Report: React.FunctionComponent<ReportProps> = () => {
+  const [taxesCountry, setTaxesCountry] = useState<Partial<CountryCode>>("fr");
+  const [residency, setResidencyCountry] = useState<Partial<CountryCode>>("fr");
+
+  const queryClient = useMemo(() => new QueryClient(), []);
+
+  return (
+    <QueryClientProvider client={queryClient}>
+      <div className="min-h-screen">
+        <header>
+          <div
+            className={classNames(
+              "w-fit flex p-4 items-center gap-2",
+              "hover:opacity-75 cursor-pointer",
+            )}
+          >
+            <div className="text-2xl font-bold">Tax Report</div>
+            <div className="text-lg">
+              <label>Residence country:</label>
+              <select
```

Comment:
do you mind moving this select to the shared/ui folder so we can easily customize the style later?